src/utils/filterTodos.ts

```diff
@@ -0,0 +1,31 @@
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+export function filterTodos(todos: Todo[], query: string, status: Status) {
+  let preparedTodos = [...todos];
```

Comment:
array is not a primitive data type, you can change it even if it will be a constant, no need to use let